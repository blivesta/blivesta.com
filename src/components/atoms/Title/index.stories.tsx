import React from 'react';
import styled from 'styled-components';
import { storiesOf } from '@storybook/react';
import Title from '.';

const Wrapper = styled.div`
  width: 480px;
  padding: 60px;
`;

storiesOf(`atoms|Title`, module).add(`default`, () => (
  <Wrapper>
    <Title>Title</Title>
  </Wrapper>
));
