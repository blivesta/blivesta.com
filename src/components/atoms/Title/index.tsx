import styled from 'styled-components';

const Title = styled.h1`
  font-family: ${({ theme }) => theme.fonts.avantGarde};
  font-size: ${({ theme }) => theme.fontSizes[4]};
  text-transform: uppercase;
  line-height: 1;
  margin-top: 0;

  ${({ theme }) => theme.media.sm`
    font-size: ${theme.fontSizes[6]};
  `}
`;

export default Title;
