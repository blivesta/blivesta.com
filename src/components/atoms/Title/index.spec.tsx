import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Title from '.';

describe('Title', () => {
  it('snapshot', () => {
    const tree = shallow(<Title>...</Title>);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
