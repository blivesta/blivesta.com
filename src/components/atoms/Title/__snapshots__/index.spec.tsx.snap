// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  font-family: ",
          [Function],
          ";
  font-size: ",
          [Function],
          ";
  text-transform: uppercase;
  line-height: 1;
  margin-top: 0;

  ",
          [Function],
          "
",
        ],
      },
      "displayName": "styled.h1",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "h1",
      "toString": [Function],
      "usesTheme": true,
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  ...
</StyledComponent>
`;
